integration-tests/tests/worker.rs

```diff
@@ -1326,6 +1326,304 @@ async fn search_oplog_1(deps: &EnvBasedTestDependencies, _tracing: &Tracing) {
     assert_eq!(result3.len(), 2); // two invocations
 }
 
+#[test]
+#[tracing::instrument]
+#[timeout(120000)]
+async fn fork_worker_1(deps: &EnvBasedTestDependencies, _tracing: &Tracing) {
```

Comment:
As there are many tests I would put them in a separate `fork.rs` (and make sure in the Makefile.toml it is listed in the integration-tests task)